examples/guides/mp-restful-webservice/README.adoc

```diff
@@ -669,223 +499,210 @@ accepting it. But omitting that validation lets us create an easy health check
+<3> If not, report that this liveness check is `down` and add an explanatory
+message.
+<4> If so, report that this liveness check is `up`.
+<5> Either way, build the response and return it.
+--
 
-// _include::0-47:{se-guide-adoc}[tags=rebuildAndRerunService;tryReadiness;tryLiveness]
+// _include::0-67:{se-guide-adoc}[tags=rebuildAndRerunService;tryReadiness;!se-HealthChecks-notes]
 === Stop, rebuild and rerun your service
 
 . Stop any running instance of your app.
 . Rebuild the app and then run it.
 
 
-=== Try the readiness check
-Access the readiness check endpoint:
+=== Check the server's health
+Run this command
 [source,bash]
-curl -i -X GET http://localhost:8080/ready
+curl -i -X GET http://localhost:8080/health
+
+and you should see output like this
+[listing,subs=+quotes]
+----
+{"outcome":"UP","checks":[{"name":"deadlock","state":"UP"},{"name":"diskSpace","state":"UP","data":{"free":"180.64 GB","freeBytes":193955860480,"percentFree":"38.79%","total":"465.72 GB","totalBytes":500068036608}},*{"name":"greetingAlive","state":"UP"}*,{"name":"heapMemory","state":"UP","data":{"free":"230.86 MB","freeBytes":242074232,"max":"4.00 GB","maxBytes":4294967296,"percentFree":"99.39%","total":"256.00 MB","totalBytes":268435456}}]}
+----
 
-Our readiness check returns no payload, just the 200 status, so you won't see any data
-displayed. The `-i` option shows the
-200 status in the response.
+The first item is `outcome` which describes the overall health of the
+server based on all the other indicators. All indicators are `UP` so the outcome
+is as well. You should also see our app-specific liveness check in the output
+(bolded above).
 
-=== Try the liveness check
-. Ping the health check endpoint
+=== Arrange for an unhealthy report
+Recall that our simple rule for liveness is that the greeting be non-null and
+non-empty. We can easily force our server to report an unhealthy state.
+
+. Set the greeting to a blank.
 +
 --
-Without changing the greeting, ping the health endpoint:
 [source,bash]
-curl -i -X GET http://localhost:8080/alive
+curl -X PUT http://localhost:8080/greet/greeting/%20
 
-The greeting is valid and in that case our health check code simply returns a 200
-with no payload.
+Our code to update the greeting accepts this and saves it as the new greeting.
 --
 
-. Set the greeting to a blank
+. Ping the health check endpoint again with the same command as before.
 +
 --
 [source,bash]
-curl -X PUT http://localhost:8080/greet/greeting/%20
+curl -i -X GET http://localhost:8080/health
 
-Our code to update the greeting accepts this and saves it as the new greeting.
+This time you should see these two parts of the output indicating that something is
+wrong:
+[listing]
+----
+{"outcome":"DOWN",...
+{"name":"greetingAlive","state":"DOWN","data":{"greeting":"not set or is empty"}}
+----
+and with the `-i` added to the `curl` command you would see status 503 "Service
+Unavailable" returned.
```

Comment:
If you include `-i` in the `curl` command, the request would return the status 503 "Service Unavailable".

Reply:
done